src/suresh/arrayExample4.java

```diff
@@ -0,0 +1,76 @@
+package suresh;
+
+public class arrayExample4 {
+
+	int listNumbersAsc() {
+		int[] num = new int[5];
+		for (int index = 0; index < num.length; index++) {
+			num[index] = index + 1;
+			System.out.println("Series of numbers:" + index);
+		}
+		return 0;
+	}
+
+	int listNumbersDesc() {
+		int[] num = new int[0];
+		for (int index = 5; index > num.length; index--) {
+			System.out.println("Length to 0:" + index);
+		}
+		return 0;
+	}
+
+	int maxNumber(int num[]) {
+		int max = num[0];
+
+		for (int index = 0; index < num.length; index++) {
+
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files